charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-daemonset.yaml

```
@@ -114,3 +135,11 @@ spec:
       - name: kernel-modules
         hostPath:
           path: /lib/modules
+      - name: kube-proxy-init-script
+        configMap:
+          name: kube-proxy-init-script
+          defaultMode: 0777
+      - name: kube-proxy-mode
+        hostPath:
+          path: /kube-proxy-mode
```

Probably we should not put in at root level but rather in some other folder. In comparison to the kubelet directory, what about

```suggestion
          path: /var/lib/kube-proxy
```